pallets/api/src/messaging/mod.rs

```diff
@@ -854,16 +845,10 @@ pub mod pallet {
 			callback: Callback,
 			id: &MessageId,
 			data: &impl Encode,
-			// This should include T::CallbackExecutor::execution_weight() where applicable unless
-			// already charged.
-			static_weight_adjustment: Option<Weight>,
 		) -> DispatchResult {
 			// This is the total weight that should be deducted from the blockspace for callback
 			// execution.
-			let max_weight = callback
-				.weight
-				.checked_add(&static_weight_adjustment.unwrap_or(Zero::zero()))
-				.ok_or(Error::<T>::BlockspaceAllowanceReached)?;
```

Comment:
this is the piece I am talking about. Should it be saved?

Reply:
This line is depricated now we have seperated the `static_weight_adjustment` and the callback weight